sync/ui.md

```diff
@@ -20,16 +26,41 @@ There are 4 main screens in the initial Admin UI:
 
 ### Data Schema
 
-x
+*TODO*: why a resolver per property (see screenshot)? Should it not be one resolver per type?
+*TODO*: is the left side editor interactive (should i be able to add/delete nodes in there)?
+
+The [Schema Editor](https://redhat.invisionapp.com/share/4RLVQKKQ2VS#/screens/305421181) will be used to define the types, mutations and subscriptions of the user's data structure. It will be split in two sections:
+
+. left side: a text editor where the user can edit the source of the schema
+. right side: an graphical (tree) representation of the schema. This will be used to explore the schema and also add a resolver to a type
 
 ### Data Sources
 
-x
+This [view](https://redhat.invisionapp.com/share/4RLVQKKQ2VS#/screens/305423609) allows the user to view and manage data sources. Many data sources can be used in a single schema. There will be a button called `Add Data Source` which opens a [Pop-up](https://redhat.invisionapp.com/share/4RLVQKKQ2VS#/screens/305424878) where the user can specify the type and credentials of the Data source.
+Editing and deleting Data sources should also be possible. 
 
 ### Resolver Mappings
 
-x
+*TODO*: Screen for resolver mappings list missing?
```

Comment:
ping @david-martin

Reply:
The screen is a little misleading.
The resolver mappings are text inputs.
The dropdown isn't in scope for now, but when it is, it will just pre-populate the text input from a set of selectable examples. The developer is free to modify it as needed.